social-messenger/src/components/CreateChannel/CreateChannel.js

```diff
@@ -154,27 +154,29 @@ const CreateChannel = ({ onClose }) => {
       <header>
         <div className='messaging-create-channel__left'>
           <div className='messaging-create-channel__left-text'>To: </div>
-          {!!selectedUsers?.length && (
-            <div className='messaging-create-channel__users'>
-              {selectedUsers.map((user) => (
-                <div className='messaging-create-channel__user' onClick={() => removeUser(user)} key={user.id}>
-                  <div className='messaging-create-channel__user-text'>{user.name}</div>
-                  <XButton />
-                </div>
-              ))}
-            </div>
-          )}
-          <form onSubmit={addUser}>
-            <input
-              autoFocus
-              ref={inputRef}
-              value={inputText}
-              onChange={(e) => setInputText(e.target.value)}
-              placeholder={!selectedUsers.length ? 'Start typing for suggestions' : ''}
-              type='text'
-              className='messaging-create-channel__input'
-            />
-          </form>
+          <div>
```

Comment:
is this div addition doing anything specific for us?

Reply:
yes, I added a className to it in the most recent version. It's the container that formats the form/input and the users div